### PR TITLE
Cfto

### PR DESCRIPTION
**Pull request by sumitroy2011**
Please merge the latest CFTO changes to master

### CODE REVIEW

File: spec/cfto/_A_add_to_bag.test.js [hunk 1]
```diff
@@ -1,111 +1,110 @@
...
+
+        browser.click('button[data-analyticsfunction="analyticsCheckoutButtonClicked"]'); // This will appear in case of basket merge scenario
+
+        cfto.chooseCollection(); // To choose the collection date & slot
+
+        // Add a new card from payment page
+        browser.click('#add-card-btn-inline');
+        cfto.cardDetailsLightBox(); // To put in the card details
+        browser.click('button[data-element="AddCardAddressOverlay"]');
+        browser.waitForVisible('#address-details__add', 10000);
```
Comment (SunilSyal) [inline spec/cfto/_A_add_to_bag.test.js]: Remove time from these calls.

File: spec/cfto/_A_add_to_bag.test.js [hunk 2]
```diff
@@ -1,111 +1,110 @@
...
+        //Choose guest checkout option on the checkout login page
+        browser.click('#guestCheckoutContButton');
+
+        // To choose collection date & slot
+        cfto.chooseCollection();
+
+        // To put in the card details
+        cfto.cardDetailsForm();
+        browser.click('.billing-form .payment__manual-link');
+        browser.selectByValue('.billing-form #personTitle', 'Mr');
```
Comment (SunilSyal) [inline spec/cfto/_A_add_to_bag.test.js]: Use functions for common code.

File: spec/cfto/_A_add_to_bag.test.js (hunk 1 above)
Comment (sumitroy2011) [reply spec/cfto/_A_add_to_bag.test.js]: Removed time from all calls

File: spec/cfto/_A_add_to_bag.test.js (hunk 2 above)
Comment (sumitroy2011) [reply spec/cfto/_A_add_to_bag.test.js]: Moved the common code into functions

**Pull request closed without merging**